source/our_tools/other_pipelines.rst

```diff
@@ -3,13 +3,33 @@
 After tractoflow
 ================
 
+.. role:: bash(code)
+   :language: bash
+
 Once the preprocessing and the tracking are done, scilpy also contains scripts to separate your data into known bundles (Recobundles), vizualize your results (ex, scil_visualize_bundles_mosaic.py), compute statistics on your bundles and so on.
 
 QA/ QC  (Quality assurance, Quality check)
 ------------------------------------------
 
-We have a QA pipeline which creates a summary of the quality of your data, you can ask Guillaume if you are interested. ToDo
+We have a QC pipeline which creates a summary of the quality of your data.
+
+*How to install it?*
+
+    The easiest is to use it as a nextflow with singularity (see :ref:`ref_nextflow` for more information.) First download the singularity image: On this page, https://github.com/GuillaumeTh/dmriqcpy/actions, click on the latest version, then under "Artifacts", click on the singularity to start the download. You should take a note somewhere to remember which version you downloaded. Then, download the nextflow:
```

Comment:
Now the singularity is available in release tab: https://github.com/GuillaumeTh/dmriqcpy/releases